### PR TITLE
Added an article for array prototype unshift

### PR DESCRIPTION
**Pull request by sujanadiga**
This article is about Javascript array prototype method `.unshift()`

### CODE REVIEW

File: JS-Array-Prototype-Unshift.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+```javascript
+var array = [1, 2, 3, 4, 5];
+
+array.unshift(0);
+// If we console.log(array.shift()); the console would output 6.
+// array is now [0, 1, 2, 3, 4, 5];
+
+array.unshift([-1]);
+// array is now [[-1], 0, 1, 2, 3, 4, 5];
+
```
Comment (atjonathan) [inline JS-Array-Prototype-Unshift.md]: Remove this line

**Comment by atjonathan** [comment]
### **LGTM :+1:**

@Rafase282, probably wanna take a look at headings etc. I still need you to educate me on the correct ones :stuck_out_tongue_closed_eyes:

File: JS-Array-Prototype-Unshift.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+```javascript
+var array = [1, 2, 3, 4, 5];
+
+array.unshift(0);
+// If we console.log(array.shift()); the console would output 6.
+// array is now [0, 1, 2, 3, 4, 5];
+
+array.unshift([-1]);
+// array is now [[-1], 0, 1, 2, 3, 4, 5];
+```
```
Comment (atjonathan) [inline JS-Array-Prototype-Unshift.md]: Add REPL Snippet

File: JS-Array-Prototype-Unshift.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+The elements to add to the front of the array.
+
+## Returns
+
+The new `length` of the array upon which the method was called.
+
+## Description
+
+The `.unshift()` method inserts the given values to the beginning of an array.
```
Comment (atjonathan) [inline JS-Array-Prototype-Unshift.md]: Do we need this after having

> The JavaScript array method `.unshift()` adds one or more elements to the beginning of an array and returns the new length of the array.

on line 3.

**Comment by Rafase282** [comment]
The markdown is fine.

File: JS-Array-Prototype-Unshift.md (hunk 3 above)
Comment (sujanadiga-practo) [inline JS-Array-Prototype-Unshift.md]: Will remove description

File: JS-Array-Prototype-Unshift.md (hunk 2 above)
Comment (sujanadiga-practo) [inline JS-Array-Prototype-Unshift.md]: Added REPL snippet

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing :tada:

**Pull request merged**